deployments/arbitrum-goerli/usdc/deploy.ts

```diff
@@ -0,0 +1,69 @@
+import { Deployed, DeploymentManager } from '../../../plugins/deployment_manager';
+import { DeploySpec, deployComet } from '../../../src/deploy';
+
+const HOUR = 60 * 60;
+const DAY = 24 * HOUR;
+
+const GOERLI_TIMELOCK = '0x8Fa336EB4bF58Cfc508dEA1B0aeC7336f55B1399';
+
+export default async function deploy(deploymentManager: DeploymentManager, deploySpec: DeploySpec): Promise<Deployed> {
+  const trace = deploymentManager.tracer()
+  const ethers = deploymentManager.hre.ethers;
+
+  // pull in existing assets
+  const USDC = await deploymentManager.existing('USDC', '0x72A9c57cD5E2Ff20450e409cF6A542f1E6c710fc', 'arbitrum-goerli');
+  const WETH = await deploymentManager.existing('WETH', '0xe39ab88f8a4777030a534146a9ca3b52bd5d43a3', 'arbitrum-goerli');
+
+  // Deploy ArbitrumBridgeReceiver
+  const bridgeReceiver = await deploymentManager.deploy(
+    'bridgeReceiver',
+    'bridges/arbitrum/ArbitrumBridgeReceiver.sol',
+    []
+  );
+
+  // Deploy Local Timelock
+  const localTimelock = await deploymentManager.deploy(
+    'timelock',
+    'vendor/Timelock.sol',
+    [
+      bridgeReceiver.address, // admin
+      1 * DAY,                // delay
+      14 * DAY,               // grace period
+      12 * HOUR,              // minimum delay
+      30 * DAY                // maxiumum delay
+    ]
+  );
+
+  // Initialize ArbitrumBridgeReceiver
+  await deploymentManager.idempotent(
+    async () => !(await bridgeReceiver.initialized()),
+    async () => {
+      trace(`Initializing BridgeReceiver`);
+      await bridgeReceiver.initialize(
+        GOERLI_TIMELOCK,      // govTimelock
+        localTimelock.address // localTimelock
+      );
+      trace(`BridgeReceiver initialized`);
+    }
+  );
+
+  // Deploy Comet
+  const deployed = await deployComet(deploymentManager, deploySpec);
+  const { comet } = deployed;
+
+  // Deploy Bulker
+  const bulker = await deploymentManager.deploy(
+    'bulker',
+    'bulkers/BaseBulker.sol',
+    [
+      await comet.governor(), // admin
+      WETH.address            // weth
+    ]
+  );
+
```

Comment:
We'll need to deploy fauceteer as well.

Reply:
that's not the case, right? we're using bridged versions of the assets from Goerli. so you can drip from the Goerli faucet and then bridge to arbitrum-goerli

Reply:
I figure we would want to mint ARB ourselves on testnet, which would require a fauceteer.

Reply:
To avoid any conflicts, I'll make this change after some of the changes from the proposal branch are brought over.

Reply:
So turns out ARB is actually a bridged token as well (unlike OP), so we would actually need to deploy ARB on Goerli and use the bridged version on Arbitrum-Goerli. Spoke with 🥈 offline and decided that we will skip on supporting ARB on testnet for now since we already test bridged assets with the other collateral assets and having to drip ARB on Goerli complicates the UI.